peachjam/js/i18n.ts

```diff
@@ -17,9 +17,14 @@ const messages: {
   [key: string] : any
 } = {};
 
-langs.forEach(key => {
-  messages[key] = loadJSONFile(`${key}/translation.json`);
-});
+// If no languages found, load en/translation.json
```

Comment:
Why not just set the default on line 5 to `["en"]`?

Reply:
done